tests/integration/redshift.spec.ts

```diff
@@ -180,8 +188,6 @@ suite("@dataform/integration/redshift", { parallel: true }, ({ before, after })
   suite("evaluate", async () => {
     test("evaluate from valid compiled graph as valid", async () => {
       const compiledGraph = await compile("tests/integration/redshift_project", "evaluate");
-      const executionGraph = await dfapi.build(compiledGraph, {}, dbadapter);
```

Comment:
I don't understand this change:
- this build & run is required so that the tests work for a "clean" redshift (otherwise the evaluated queries fail, because the relevant tables don't exist)
- this shouldn't be executing the same run twice, by virtue of the `evaluate` schema suffix passed in on line 182 (all of the independent runs in this file should be using a different schema suffix to avoid exactly that problem)

Reply:
Sorry should have read this in more depth.

I've filed another bug #918, for now I've turned parallelism off which also solves the problem.